tests/python/gaia-ui-tests/gaiatest/tests/brick/test_brick_verification.py

```diff
@@ -0,0 +1,19 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from gaiatest import GaiaTestCase
+from gaiatest.apps.homescreen.app import Homescreen
+
+
+class TestBrickVerification(GaiaTestCase):
+
+    def setUp(self):
+        # Open Firefox OS and Skip the First Time Use app
+        GaiaTestCase.setUp(self)
+        # Open the homescreen app
+        self.homescreen = Homescreen(self.marionette)
```

Comment:
This doesn't open the homescreen app, it just instantiates an app object. I wouldn't bother putting this into a setUp method because there's nothing really unique about this override.